README.md

```diff
@@ -8,12 +8,13 @@
 [![Monthly Downloads](https://poser.pugx.org/symfony-cmf/routing-bundle/d/monthly)](https://packagist.org/packages/symfony-cmf/routing-bundle)
 [![Daily Downloads](https://poser.pugx.org/symfony-cmf/routing-bundle/d/daily)](https://packagist.org/packages/symfony-cmf/routing-bundle)
 
-Version/Branch | Travis | Coveralls |
------- | ------ | --------- |
-2.1.1   | [![Build Status][travis_stable_badge]][travis_link]     | [![Coverage Status][coveralls_stable_badge]][coveralls_stable_link]     |
-master | [![Build Status][travis_unstable_badge]][travis_link] | [![Coverage Status][coveralls_unstable_badge]][coveralls_unstable_link] |
+Branch | Travis | Coveralls | Scrutinizer |
+------ | ------ | --------- | ----------- |
+2.1   | [![Build Status][travis_stable_badge]][travis_stable_link]     | [![Coverage Status][coveralls_stable_badge]][coveralls_stable_link]     | [![Scrutinizer Status][scrutinizer_stable_badge]][scrutinizer_stable_link] |
+3.0-dev | [![Build Status][travis_unstable_badge]][travis_unstable_link] | [![Coverage Status][coveralls_unstable_badge]][coveralls_unstable_link] | [![Scrutinizer Status][scrutinizer_unstable_badge]][scrutinizer_unstable_link] |
```

Comment:
there is no branch 3.0-dev, and master represents 2.2. i'd keep master here and never point to upcoming not-yet-master branches as the dev version.

Reply:
when requiring `symfony-cmf/routing-bundle:3.0-dev` we should get master now, when this PR is merged, isn't it?

Reply:
you are not changing the branch-alias in composer.json so there will not be a 3.0-dev version.

Reply:
and i would keep this at `master` so its whatever master is at any time. one less thing that we can forget to update :-)